### ISSUE TITLE
Handle Login

### ISSUE THREAD

**Issue by onionmansz** [SvetoslavDoychinov/yuumi-bot#1]
Would you handle login?

**Comment by SvetoslavDoychinov** [comment]
Yes, I will but probably next week. Let the bot play with a friend today and we found some bugs on the Yuumi behaviours. Once I fix those for good will start working on login.

**Comment by onionmansz** [comment]
Can share endpoints if needed.

Also should add random champ from free game rotation to play intro
Sent from my iPhone

On Apr 10, 2024, at 10:12 PM, Svetlio02 ***@***.***> wrote:

﻿

Yes, I will but probably next week. Let the bot play with a friend today and we found some bugs on the Yuumi behaviours. Once I fix those for good will start working on login.

**Comment by dodag888** [comment]
Yumi becomes fluid, heals.
And staying still in the well
